dcc-portal-ui/app/scripts/releases/styles/_home.scss

```diff
@@ -373,9 +373,12 @@
     }
   }
 
+  .before-updates {
+    margin-bottom: $updates-offset - $spacing-2x;
```

Comment:
Compensates for the negative margin the updates use to overflow from the grey row